cosmwasm/submit-proposal.js

```diff
@@ -161,6 +161,41 @@ const execute = async (client, wallet, config, options) => {
     await callSubmitProposal(client, wallet, config, options, proposal);
 };
 
+const registerItsChain = async (client, wallet, config, options) => {
+    const chains = options.chains.map((chain) => {
+        const chainConfig = getChainConfig(config, chain);
+
+        if (!chainConfig.maxUintBits) {
+            throw new Error(`Missing maxUintBits for chain ${chain}`);
+        }
+
+        if (!chainConfig.maxDecimalsWhenTruncating) {
+            throw new Error(`Missing maxDecimalsWhenTruncating for chain ${chain}`);
+        }
+
+        const itsEdgeContract = chainConfig.contracts.InterchainTokenService?.address || chainConfig.contracts.ITS?.objects?.ChannelId;
+
+        if (!itsEdgeContract) {
+            throw new Error(`Missing ITS edge contract for chain ${chain}`);
+        }
+
+        return {
+            chain: chainConfig.axelarId,
+            its_edge_contract: itsEdgeContract,
+            truncation: {
+                max_uint: (2n ** BigInt(chainConfig.maxUintBits) - 1n).toString(),
+                max_decimals_when_truncating: chainConfig.maxDecimalsWhenTruncating,
+            },
+        };
+    });
+
+    await execute(client, wallet, config, {
```

Comment:
does this print the proposal with a prompt before submitting for verification?

Reply:
yes it does